.circleci/unittest/linux/scripts/run_test.sh

```diff
@@ -15,8 +15,16 @@ python -m torch.utils.collect_env
 export TORCHAUDIO_TEST_FAIL_IF_NO_EXTENSION=1
 export PATH="${PWD}/third_party/install/bin/:${PATH}"
 
+declare -a common_args=(
+    '--import-mode' 'importlib'
```

Comment:
See https://docs.pytest.org/en/stable/pythonpath.html for the detail of the behavior change by `--import-mode`.

By default `pytest` modifies `sys.path` at runtime and prepend the path to the root directory of the repository, and it cause the source directory of `torchaudio` shadows the version of `torchaudio` built/installed instead. We want to test the installed version so we do not want `pytest` to modify `sys.path`.